open62541 build broken

https://oss-fuzz-build-logs.storage.googleapis.com/index.html

@Pro - can you please take a look. I did an attempt with https://github.com/google/oss-fuzz/commit/90f98631d8badf4bf6a934823ec0b4db72772744, but looks like there are more dependencies.

@Pro - thanks for spotting that which fixed that, now it is failing here on some deprecated calls.

```
I  ERROR: build step 4 "gcr.io/oss-fuzz/open62541" failed: exit status 2 
I  ERROR 
I  Finished Step #4 
I  Step #4: make: *** [all] Error 2 
I  Step #4: Makefile:149: recipe for target 'all' failed 
I  Step #4: CMakeFiles/Makefile2:135: recipe for target 'CMakeFiles/open62541-plugins.dir/all' failed 
I  Step #4: make[1]: *** [CMakeFiles/open62541-plugins.dir/all] Error 2 
I  Step #4: make[2]: *** [CMakeFiles/open62541-plugins.dir/plugins/ua_securitypolicy_basic128rsa15.c.o] Error 1 
I  Step #4: CMakeFiles/open62541-plugins.dir/build.make:270: recipe for target 'CMakeFiles/open62541-plugins.dir/plugins/ua_securitypolicy_basic128rsa15.c.o' failed 
I  Step #4: 3 errors generated. 
I  Step #4:                                                ^ 
I  Step #4: #define MBEDTLS_DEPRECATED      __attribute__((deprecated)) 
I  Step #4: /usr/include/mbedtls/sha1.h:302:48: note: expanded from macro 'MBEDTLS_DEPRECATED' 
I  Step #4: ^ 
I  Step #4: MBEDTLS_DEPRECATED static inline void mbedtls_sha1( const unsigned char *input, 
I  Step #4: /usr/include/mbedtls/sha1.h:320:1: note: 'mbedtls_sha1' has been explicitly marked deprecated here 
I  Step #4:     ^ 
I  Step #4:     mbedtls_sha1(certificate->data, certificate->length, thumbprint->data); 
I  Step #4: /src/open62541/plugins/ua_securitypolicy_basic128rsa15.c:265:5: error: 'mbedtls_sha1' is deprecated [-Werror,-Wdeprecated-declarations] 
I  Step #4:                                                ^ 
I  Step #4: #define MBEDTLS_DEPRECATED      __attribute__((deprecated)) 
I  Step #4: /usr/include/mbedtls/sha1.h:302:48: note: expanded from macro 'MBEDTLS_DEPRECATED' 
I  Step #4: ^ 
I  Step #4: MBEDTLS_DEPRECATED static inline void mbedtls_sha1( const unsigned char *input, 
I  Step #4: /usr/include/mbedtls/sha1.h:320:1: note: 'mbedtls_sha1' has been explicitly marked deprecated here 
I  Step #4:     ^ 
I  Step #4:     mbedtls_sha1(message->data, message->length, hash); 
I  Step #4: /src/open62541/plugins/ua_securitypolicy_basic128rsa15.c:113:5: error: 'mbedtls_sha1' is deprecated [-Werror,-Wdeprecated-declarations] 
I  Step #4:                                                ^ 
I  Step #4: #define MBEDTLS_DEPRECATED      __attribute__((deprecated)) 
I  Step #4: /usr/include/mbedtls/sha1.h:302:48: note: expanded from macro 'MBEDTLS_DEPRECATED' 
I  Step #4: ^ 
I  Step #4: MBEDTLS_DEPRECATED static inline void mbedtls_sha1( const unsigned char *input, 
I  Step #4: /usr/include/mbedtls/sha1.h:320:1: note: 'mbedtls_sha1' has been explicitly marked deprecated here 
I  Step #4:     ^ 
I  Step #4:     mbedtls_sha1(message->data, message->length, hash); 
I  Step #4: /src/open62541/plugins/ua_securitypolicy_basic128rsa15.c:90:5: error: 'mbedtls_sha1' is deprecated [-Werror,-Wdeprecated-declarations] 
I  Step #4: [ 14%] Building C object CMakeFiles/open62541-plugins.dir/plugins/ua_securitypolicy_basic128rsa15.c.o 
```

--- a/projects/open62541/Dockerfile
+++ b/projects/open62541/Dockerfile
@@ -18,10 +18,10 @@ FROM gcr.io/oss-fuzz-base/base-builder
 MAINTAINER git@s.profanter.me
 RUN apt-get update && apt-get install -y make cmake python-six wget
 # We need libmbedtls > 2.5.1 otherwise it does not include the lib for static linking
-RUN wget http://ftp.us.debian.org/debian/pool/main/m/mbedtls/libmbedtls-dev_2.7.0-2_amd64.deb && \
-    wget http://ftp.us.debian.org/debian/pool/main/m/mbedtls/libmbedcrypto0_2.6.0-1_amd64.deb && \
-    wget http://ftp.us.debian.org/debian/pool/main/m/mbedtls/libmbedtls10_2.7.0-2_amd64.deb && \
-    wget http://ftp.us.debian.org/debian/pool/main/m/mbedtls/libmbedx509-0_2.7.0-2_amd64.deb && \
+RUN wget https://open62541.org/libmbedtls/libmbedtls-dev_2.6.0-1_amd64.deb && \
+    wget https://open62541.org/libmbedtls/libmbedcrypto0_2.6.0-1_amd64.deb && \
+    wget https://open62541.org/libmbedtls/libmbedtls10_2.6.0-1_amd64.deb && \
+    wget https://open62541.org/libmbedtls/libmbedx509-0_2.6.0-1_amd64.deb && \
     dpkg -i *.deb
 RUN git clone --depth 1 https://github.com/open62541/open62541.git -bmaster open62541
 WORKDIR open62541

